README.rst

```diff
@@ -13,3 +13,131 @@ __ https://discuss.openedx.org/c/working-groups/build-test-release/30
+
+*Current assignee: @arbrandes*
+
+The group chair is responsible for coordinating efforts that culminate in
+getting a new release out and maintaining the current one.  In particular, the
+Group Chair will:
+
+* Manage the current release cycle in general, including making sure that
+  proper deadlines are established and adhered to
+* Facilitate the assignment of roles and tasks
+* Coordinate the group's communication efforts, including running periodic
+  meetings such as the Biweekly Meetup, and summarizing recent happenings in
+  the Biweekly Update
+* Try and help solve problems as they come up, bringing them to the attention
+  of the right group members
+
+Release Manager
+===============
+
+*Current assignees: @nedbat and @arbrandes*
+
+The Release Manager is responsible for the actual cutting of a release, which,
+in essence, includes following the steps in the `Process to Create an Open edX Release
+<https://openedx.atlassian.net/wiki/spaces/COMM/pages/19662426/Process+to+Create+an+Open+edX+Release>`_.
+
+In addition, the Release Manager will:
+
+* Create public test or release candidate branches as required by the release process
+* Merge reviewed PRs into the release master branch (for instance,
+  ``koa.master``)
+* Notify the Release Testing Coordinator whenever additional testing is needed
+  in response to the above
+* Tag releases, major and minor, at the appropriate time
+
+Release Testing Coordinator
+===========================
+
+*Current assignee: @jfavellar90*
+
+The Release Testing Coordinator is responsible for making sure release
+candidates are tested _before_ a tag is made.  The coordinator does not
+necessarily have to run tests themselves, but they will:
+
+* Coordinate testing in general, in particular in the days leading to a timed
+  release, including the bimonthly minor ones.
+* Keep tabs on whatever CI is available, making sure it is running correctly
+* Advocate for better testing tools and environments, when necessary
+* Coordinate the reaction to test failures, such as bringing them to the
+  attention of the rest of the group by opening tickets or commenting in the forum
+
+Release Documentation Expert
+============================
+
+*Current assignee: N/A*
+
+The Release Documentation Expert is tasked with building the Release Notes
+document prior to a major release.  It involves:
+
+* Tracking down all relevant information,
+* Writing a draft document and posting it for community review with
+sufficient lead time prior to a release
+* Adjusting it as per the former review, and finally releasing a final version
+  concomittantly with the major release itself
+
+Bug Triager
+===========
+
+*Current assignee: @BbrSofiane*
+
+The group's Bug Triager is, in a nutshell, responsible for proper use of
+the group's `issue board
+<https://github.com/openedx/build-test-release-wg/projects/1>`_.  In
+particular, they will:
+
+* Tag incoming issues with the latest release that is still affected (i.e.,
```

Comment:
Can this be rephrased to 'Review tags on incoming issues and ensure it has been tagged with the latest release that is still affect''? 

Just to clarify, If someone raises the issue in say Juniper, it should be the responsibility of the assignee to test whether it affects koa and update the tag accordingly. Otherwise, the bug triager might have to test a lot of issues.

Reply:
Also, BTW, edX has never supported older releases.  Once Koa is out, Juniper is unsupported.  We don't have to keep that rule, but that's the way it's been done in the past.

Reply:
If we decided to only support the latest release, we might end up creating a gap with most of the community as they might not upgrade at the same pace until we have an easier upgrade process. I would be in favour of supporting one release behind. Maybe how we define 'support' for older releases can be slightly different?

Reply:
One thing to consider here is that if we start supporting older releases, it will further disencentivize people from upgrading from older releases. 

At the same time, the fact that we currently say "only the last release is supported" means that we can't highlight any "Doomsday countdown", like the end of the support coming up for a Python, Django or Ubuntu release. We are basically saying that anyone not instantly upgrading to the latest release is already doomed... Which might not be perceived as realistic, and thus more ignored?

Reply:
@BbrSofiane, I'm fine with rewording the tagging, but I think @nedbat has a better point below: this is too specific for this document, right now.  It's better we discuss it on your ADR, so I've reworded it accordingly here.
 
On release support: while supporting any release other than the last one would be a generally good thing to do, I don't think the group currently has the capacity to do so.  Not only would we need more resources, we would need to get better at organizing them.  And I don't think we're there, yet.

But it's certainly something that's worth starting a conversation about.  (@antoviaque's points above are all good ones, for instance.)  I suggest that we start doing so after Lilac, with a proper ADR to ground the discussion.  What do you think?